crates/viewer/re_selection_panel/src/visualizer_ui.rs

```diff
@@ -279,8 +280,17 @@ fn visualizer_components(
                     }
                 };
 
+                // TODO(#6889): query results should already operate on full descriptors
+                let component_descr = db
+                    .storage_engine()
+                    .store()
+                    .entity_component_descriptors_with_name(&entity_path, component_name)
+                    .into_iter()
+                    .next()
+                    .unwrap_or_else(|| ComponentDescriptor::new(component_name));
```

Comment:
Do you think it's worth to `warn!` here?

Reply:
yeah let's do that. should never happen and if it does while we're working on this we'd really want to know